.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$config = \TYPO3\CodingStandards\CsFixerConfig::create();
+$config
+    ->getFinder()->in(__DIR__)->exclude([__DIR__ . '/.Build', __DIR__ . '/.ddev'])
```

Comment:
```suggestion
    ->getFinder()
    ->in(__DIR__)
    ->exclude([
        __DIR__ . '/.Build',
        __DIR__ . '/.ddev'
    ])
```

Improves readability a lot.